tools/sst_dump_tool.cc

```diff
@@ -588,6 +593,17 @@ int SSTDumpTool::Run(int argc, char** argv) {
       }
     }
 
+    if (command == "verify") {
```

Comment:
sorry I didn't understand, I can run `./sst_dump --command=check --verify_checksum --file=...` and it'll verify checksums once as a command.